async_asgi_testclient/testing.py

```diff
@@ -158,8 +163,10 @@ def websocket_connect(self, path, extra_headers=None):
         input_queue: asyncio.Queue[dict] = asyncio.Queue()
         output_queue: asyncio.Queue[dict] = asyncio.Queue()
 
+        if headers is not None:
+            self.headers.update(headers)
```

Comment:
Sorry, I think my comment https://github.com/vinissimus/async-asgi-testclient/issues/19#issuecomment-546525951 was ambiguous. My idea is to merge the headers provided in open() method with the global headers (but without modifying the global headers). 

I meant to say something like
```python
merged_headers = self.headers.copy()
merged_headers.update(headers)
...
make_test_headers_oath_query_string(..., merged_headers)
```

Reply:
Ahh I see, I'll change it